apps/client/src/mocks/Apis/product/productApi.ts

```diff
@@ -0,0 +1,68 @@
+import { rest } from "msw";
+import { productMockData } from "./productMock";
+
+export type ProductType = {
+  storeIdx: number;
+  storeName: string;
+  description: string;
+  thumbnailUrl: string;
+  latLng: {
+    lat: number;
+    lng: number;
+  };
+  isCouponAvailable: boolean;
+  isRecommend: boolean;
+  distance: number;
+  price: number;
+};
+
+export const getProductApi = rest.get<ProductType[]>("/api/products", (req, res, ctx) => {
+  const searchParams = new URLSearchParams(req.url.searchParams);
+
+  const _pageNum = searchParams.get("pageNum");
+
+  const pageNum = _pageNum ? parseInt(_pageNum, 10) : 1;
```

Comment:
parseInt의 두 번째 인자로 10을 넘겨주지 않아도 default가 10진수여서 그냥 parseInt(_pageNum)쓰시면 될 거 같아요

```suggestion
  const pageNum = parseInt(_pageNum, 10) || 1
```
으로 작성해도 의도하신 "falsy 값일 때 1로 해줘"가 가능할 것 같아요~

Reply:
넵 ! 감사합니다. 👍

Reply:
도은님 lint 규칙 때문에 안되는 것 같아요 ㅠㅠ lint가 10을 넣거나 radix disable 선언하라는 경고를 주네요.

일단 10을 넣어 놓겠습니다. :)